packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -1105,6 +1111,11 @@ export class SnapController extends BaseController<
       `${controllerName}:getFile`,
       async (...args) => this.getSnapFile(...args),
     );
+
+    this.messagingSystem.registerActionHandler(
+      `${controllerName}:stopAllSnaps`,
+      async (...args) => this.stopAllSnaps(...args),
```

Comment:
We should test this action and the accompanying function

Reply:
Done here: [1e50e98](https://github.com/MetaMask/snaps/pull/2674/commits/1e50e98360f9340c22aebc049be75fe33fe5d596)